### PR TITLE
add settings for auto resize view

### PR DESCRIPTION
**Pull request by mauroalexandre**
create a setting to fuxa auto resize view when window size is changed

### CODE REVIEW

**Comment by unocelli** [comment]
Thank you very much for the PR

File: client/src/app/_helpers/utils.ts [hunk 1]
```diff
@@ -296,6 +296,19 @@ export class Utils {
...
+    static resizeView = (selector) => {
+        document.querySelectorAll(selector).forEach((scaled: any) => {
+            let parent = scaled.parentNode,
+            ratio = (parent.offsetWidth / scaled.offsetWidth),
+            padding = scaled.offsetHeight * ratio;
+      
+          scaled.style.transform = 'scale(' + ratio + ')';
+          scaled.style.transformOrigin = 'top left';
+      
+          parent.style.paddingTop = padding;
```
Comment (unocelli) [inline client/src/app/_helpers/utils.ts:309]: Hi, why you set the paddingTop?

File: client/src/app/_helpers/utils.ts [hunk 2]
```diff
@@ -296,6 +296,19 @@ export class Utils {
...
         }, {});
     }
+
+    static resizeView = (selector) => {
+        document.querySelectorAll(selector).forEach((scaled: any) => {
+            let parent = scaled.parentNode,
+            ratio = (parent.offsetWidth / scaled.offsetWidth),
+            padding = scaled.offsetHeight * ratio;
+      
+          scaled.style.transform = 'scale(' + ratio + ')';
```
Comment (unocelli) [inline client/src/app/_helpers/utils.ts:306]: @mauroalexandre suggestion:
```
            let parent = scaled.parentNode,
            ratioWidth = (parent.offsetWidth / scaled.offsetWidth),
            ratioHeight = (parent.offsetHeight / scaled.offsetHeight);
      
            scaled.style.transform = 'scale(' + Math.min(ratioWidth, ratioHeight) + ')';
```

File: client/src/app/_helpers/utils.ts (hunk 1 above)
Comment (mauroalexandre) [reply client/src/app/_helpers/utils.ts:309]: Sorry, I need to remember, but, I think your solution is better.

File: client/src/app/_helpers/utils.ts (hunk 2 above)
Comment (mauroalexandre) [reply client/src/app/_helpers/utils.ts:306]: Nice!

**Pull request merged**